app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="int-container">
+<h2>User Registration</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+  <%= f.label :name %><br />
+  <%= f.text_field :name %>
+</div>
+<br>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+  <br>
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em><br>
+    <% end %>
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+  <br>
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+  <br>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you also add a `textarea` field for `Bio` input in the `Sign Up` form to help a user to be registered with all needed information